lldb/source/Utility/RealpathPrefixes.cpp

```diff
@@ -54,7 +53,7 @@ RealpathPrefixes::ResolveSymlinks(const FileSpec &file_spec) {
       LLDB_LOGF(log, "Realpath'ing support file %s", file_spec_path.c_str());
 
       // One prefix matched. Try to realpath.
-      llvm::SmallString<PATH_MAX> buff;
+      llvm::SmallString<1024> buff;
```

Comment:
Should we give this a smaller initial value? Looks like most `SmallString`s in LLDB use either 64, 128 or 256. Maybe we should unify this and have a `lldb::PathSmallString` that typedefs to a reasonable value.

Reply:
It's a good callout for code to share the same small initial value by defining a common constant.

But I think it's your call as to what value this should be (64, 128 or 256?). I can add the constant and use it here once you decide.

I can search the code base and see what is the most common value used and update the patch with this value as a recommendation.